content/docs/get-started/aws/modify-program.md

```diff
@@ -12,176 +12,153 @@ menu:
 aliases: ["/docs/quickstart/aws/modify-program/"]
 ---
 
-Now that we have an instance of our Pulumi program deployed, let's enable encryption on our S3 bucket.
+Now that your S3 Bucket is provisioned, let's add an object to your bucket. First let's create a new directory called `site`.
```

Comment:
```suggestion
Now that your S3 bucket is provisioned, let's add an object to it. First, create a new directory called `site`.
```

ETOOMANYLETSES. 😄 Seriously though, I think the occasional well-placed "let's" is fine, but too many starts to pull things back awkwardly back into "we" territory. You can generally lean on the imperative to keep things moving ("Modify the program" rather than "Let's modify", etc.) as it implies "you".